src/status_im/contexts/settings/wallet/events.cljs

```diff
@@ -53,3 +54,86 @@
      handle-connection)))
 
 (rf/reg-event-fx :wallet/get-key-pair-export-connection get-key-pair-export-connection)
+
+(defn extract-keypair-name
+  [db key-uids-set]
+  (when (= (count key-uids-set) 1)
+    (let [key-uid  (first key-uids-set)
+          keypairs (get-in db [:wallet :keypairs])]
+      (->> (filter #(= (:key-uid %) key-uid) keypairs)
+           first
+           :name))))
+
+(defn update-accounts
+  [accounts key-uids-set]
+  (into {}
+        (map (fn [[k account]]
+               (if (and (contains? key-uids-set (:key-uid account))
+                        (= (:operable account) :no))
+                 [k (assoc account :operable :fully)]
+                 [k account]))
+             accounts)))
+
+(defn update-keypairs
+  [keypairs key-uids-set]
+  (map (fn [keypair]
+         (if (contains? key-uids-set (:key-uid keypair))
+           (update keypair
+                   :accounts
+                   (fn [accounts]
+                     (map (fn [account]
+                            (if (and (contains? key-uids-set (:key-uid account))
+                                     (= (:operable account) ":no"))
+                              (assoc account :operable ":fully")
```

Comment:
I think the strings should be `"no"` and `"fully"`, but maybe we can also update the logic for storing the key-pair to automatically to turn them into the keywords `:no` and `:fully`. Thoughts?

Reply:
Yeah i wanted to do it but I thought it may have some side effect on other places we used it, no?

Reply:
Yeah most likely will affect other code, but when I search the codebase I can't find any references to `"fully"` or `:operable` (except the stuff we've been recently working on), so maybe it's safe to change